src/Telemetry/Opt_In/Status.php

```diff
@@ -67,44 +64,38 @@ public function get_option() {
 	 *
 	 * @since 1.0.0
 	 * @since 2.0.1 Correct logic so it is not subject to the order of the plugins.
+	 * @since TBD   Update to remove unnecessary "mixed" status.
 	 *
-	 * @return integer The status value.
+	 * @return bool The site's current opt in status.
 	 */
 	public function get() {
-		$option = $this->get_option();
 
-		// If the status option is not an option, default to inactive.
-		if ( ! isset( $option['plugins'] ) ) {
-			return self::STATUS_INACTIVE;
-		}
+		$status  = false;
+		$option  = $this->get_option();
+		$plugins = isset( $option['plugins'] ) ? $option['plugins'] : [];
 
-		$status = array_reduce(
-			$option['plugins'],
-			function( $carry, $item ) {
-				// First run, ignore the default STATUS_ACTIVE.
-				if ( empty( $carry ) ) {
-					return (int) $item['optin'];
-				}
+		if ( count( $plugins ) === 0 ) {
+			$status = false;
+		}
 
-				// As long as they are the same, we keep returning the same.
-				if ( $carry === $item['optin'] ) {
-					return (int) $item['optin'];
-				}
+		foreach ( $plugins as $plugin ) {
 
-				return self::STATUS_MIXED;
+			// If any plugins are missing an optin status or at least one is false, set status to false.
+			if ( ! isset( $plugin['optin'] ) || false === $plugin['optin'] ) {
+				$status = false;
+				break;
 			}
-		);
 
-		if ( 0 === $status ) {
-			$status = self::STATUS_INACTIVE;
+			$status = true;
 		}
 
 		/**
 		 * Filters the opt-in status value.
 		 *
 		 * @since 1.0.0
+		 * @since TBD Update to return a bool status.
 		 *
-		 * @param integer $status The opt-in status value.
+		 * @param bool $status If the site is currently opted in.
```

Comment:
Same as above, if we change this to a bool, it is a breaking change. Might be good to check with Stellar devs to make sure that isn't going to cause a problem.